toplevel/topdirs.ml

```diff
@@ -447,6 +446,13 @@ let _ = add_directive "ppxs_clear"
              (external and in-process)";
     }
 
+let _ = add_directive "remove_in_process_ppxs"
+    (Directive_none Pparse.clear_in_process_ppx)
+    {
+      section = section_options;
+      doc = "Forget all added in process PPX preprocessors";
+    }
```

Comment:
@Kakadu Just compare them by-value. Though, path is also acceptable, and might be more convenient in some cases.